src/ImageSharp/Processing/Processors/Transforms/Resize/ResamplerExtensions.cs

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Runtime.CompilerServices;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.Memory;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Processing.Processors.Transforms
+{
+    /// <content>
+    /// Extensions for <see cref="IResampler"/>.
+    /// </content>
+    public static partial class ResamplerExtensions
```

Comment:
We should also try hiding these classes from the public, while keeping extensibility.

Reply:
See above.